tests/sfko/sfko/check.py

```diff
@@ -0,0 +1,211 @@
+import base64
+import math
+import time
+from pprint import pprint
+
+from .constant import BackendType
+from .register import register_check
+from .util.log import Log
+from .util.proxy import AWS_REGEX, GCP_REGEX
+from .util.redis import (count_pending_replication, get_pending_replication,
+                         get_requests, get_requests_by_method, get_requests_by_tag, filter_requests, start_after)
+
+_log = Log('check')
+
+# Convenience function for dealing with Request objects
+
+def has_status(requests, code = 200):
+    '''
+    Takes a list of Requests, and optionally a status code
+    Returns a bool indicating if any of them have status == code
+    '''
+    for req in requests:
+        # print(req)
+        if req.status == code:
+            return True
+    return False
+
+def _check_backend_key(bucket, key, cloud):
+    delete_key, requests = get_requests(bucket, key, cloud)
+    passed = True
+    if not requests:
```

Comment:
All the block could be replaced by a more pythonic :snake: one liner which could be more readable, something like:

```python
passed = requests and any(req.status == 200 for req in requests)
```